### ISSUE TITLE
Cannot redeclare $post

### ISSUE THREAD

**Issue by Parkbyounghyo** [VemtoOrg/vemto2-issues#177]
**Error Message:**
(error) FAILED to execute command: "php" -d display_errors=on "C:\Program Files\Vemto\resources\static\VMTTL1"
    |> 
Fatal error: Cannot redeclare App\Livewire\Dashboard\Posts\Forms\UpdateDetailForm::$post in D:\laragon\www\aitest\app\Livewire\Dashboard\Posts\Forms\UpdateDetailForm.php on line 18

**Stack:**
```
Error: Command failed: "php" -d display_errors=on "C:\Program Files\Vemto\resources\static\VMTTL1"

    at ChildProcess.exithandler (node:child_process:407:12)
    at ChildProcess.emit (node:events:527:28)
    at maybeClose (node:internal/child_process:1092:16)
    at ChildProcess._handle.onexit (node:internal/child_process:302:5)
```

**How to reproduce:**
1. 
2. 
3. 

**Expected behavior:**

**Actual behavior:**

**Additional context:**

---

Vemto 2.0.23
win32 10.0.19045
Locale: ko
App version: 2.0.23

**Comment by TiagoSilvaPereira** [comment]
@Parkbyounghyo it seems you have a form with duplicated variables (maybe due to a recursive relationship). Please change the variable names to avoid this problem (I'll fix this in a future version)

**Comment by Parkbyounghyo** [comment]
my problem was wrong the table name of datebase.
table name : posts

**Comment by TiagoSilvaPereira** [comment]
Fixed on 2.0.24 for new projects. For existing projects, it is necessary to fix the wrong code manually.
